test/__mocks__/validation.mock.ts

```diff
@@ -36,7 +37,7 @@ const PRINCIPAL_ADDRESS_WITH_INVALID_CHARACTERS_FIELDS_MOCK = {
   principal_address_line_2: "площадь",
   principal_address_town: "Москва",
   principal_address_county: "Москва",
-  principal_address_country: NO_MAX,
+  principal_address_country: "Москва",
```

Comment:
This is unnecessary as there's no validation on the country field

Reply:
(except it being empty)

Reply:
Agreed. Will change this back.